lib/configure-map.js

```diff
@@ -135,10 +135,24 @@ module.exports = function (original, length, options) {
 			return conf.delete(arg);
 		};
 	}
+	extGet = function () {
+		var id, args = arguments;
+		if (resolve) args = resolve(args);
+		id = get(args);
+		return cache[id];
+	};
+	extHas = function () {
+		var id, args = arguments;
+		if (resolve) args = resolve(args);
+		id = get(args);
+		return conf.has(id);
```

Comment:
Let's exit early (before `conf.has(..)`) with `false` if `id` resolved to `null` (It can happen, e.g. https://github.com/medikoo/memoizee/blob/master/normalizers/get-1.js#L10 )